File: ar4.py
#ROTATE AN ARRAY  TO THE LEFT  BY X th POSTIONS
import numpy as np
arr=np.array([12,34,4,56,78])
x=int(input("ENTER INDEX:"))
x=x % len(arr)
rotated_array=np.roll(arr,-x) #"-VE" for left side rotation
print(rotated_array)

File: ar4.c
//ROTATE AN ARRAY TO THE LEFT BY X th POSITIONS.
#include<stdio.h>
int main(){
    int arr[]={10,20,30,40,50,60};
    int arr2[100];
    int n=sizeof(arr)/4;
    int x;
    
    printf("ENTER THE INDEX:");
    
    scanf("%d",&x);
    x=x % n ;
    for(int i=0;i<n;i++){
        arr2[i]=arr[(x+i) % n];
    }
    printf("ARRAY AFTER ROTATION:\n");
    for(int i=0;i<n;i++){
        printf("%d ",arr2[i]);
    }
    printf("\n");

}

